py/torch_tensorrt/dynamo/runtime/_TorchTensorRTModule.py

```diff
@@ -125,6 +126,7 @@ def __init__(
         self.output_binding_names = (
             output_binding_names if output_binding_names is not None else []
         )
+        self.output_shapes = output_shapes
```

Comment:
Same thing here?

Reply:
This can be a function on the engine that pulls the data from TRT

Reply:
do you mean I should add:
```
    def get_output_shapes(self) -> List[Tuple[int]]:
        return self.output_shapes
    
    def set_output_shapes(self, output_shapes: List[Tuple[int]]) -> None:
        self.output_shapes = output_shapes
```

Instead of using trt_module.output_shapes 
I should use:  trt_module.get_output_shapes()

Reply:
no longer required. now all the output shape and dtype info is in the graph node.meta['val']